infra/launch.py

```diff
@@ -158,7 +158,7 @@ def _default_run_id():
     cli.add_arg(parser, config, ["--github_token"], type=str)
 
     parser.add_argument(
-        "-e", "--env", action="append", nargs=2, metavar=("KEY", "VALUE"), default=config.get("env", {}).items()
```

Comment:
If I pass `--env` like:

```
python infra/launch.py --retries=$RETRIES --foreground --tpu_name $TPU_NAME --tpu_type $TPU_TYPE --zone $TPU_ZONE --run_id $WANDB_RUN_ID \
    --env LIBTPU_INIT_ARGS "--xla_tpu_enable_data_parallel_all_reduce_opt=true --xla_tpu_data_parallel_opt_different_sized_ops=true --xla_tpu_enable_async_collective_fusion=true --xla_tpu_enable_async_collective_fusion_fuse_all_gather=true --xla_tpu_enable_async_collective_fusion_multiple_steps=true --xla_tpu_overlap_compute_collective_tc=true --xla_enable_async_all_gather=true" \
    -- $CMD
```

This will cause:

```
Traceback (most recent call last):
  File "/Users/ivan/dev/levanter/infra/launch.py", line 165, in <module>
    args = parser.parse_args()
  File "/Users/ivan/dev/miniconda3/envs/levanter/lib/python3.10/argparse.py", line 1833, in parse_args
    args, argv = self.parse_known_args(args, namespace)
  File "/Users/ivan/dev/miniconda3/envs/levanter/lib/python3.10/argparse.py", line 1866, in parse_known_args
    namespace, args = self._parse_known_args(args, namespace)
  File "/Users/ivan/dev/miniconda3/envs/levanter/lib/python3.10/argparse.py", line 2079, in _parse_known_args
    start_index = consume_optional(start_index)
  File "/Users/ivan/dev/miniconda3/envs/levanter/lib/python3.10/argparse.py", line 2019, in consume_optional
    take_action(action, args, option_string)
  File "/Users/ivan/dev/miniconda3/envs/levanter/lib/python3.10/argparse.py", line 1943, in take_action
    action(self, namespace, argument_values, option_string)
  File "/Users/ivan/dev/miniconda3/envs/levanter/lib/python3.10/argparse.py", line 1036, in __call__
    items = _copy_items(items)
  File "/Users/ivan/dev/miniconda3/envs/levanter/lib/python3.10/argparse.py", line 147, in _copy_items
    return copy.copy(items)
  File "/Users/ivan/dev/miniconda3/envs/levanter/lib/python3.10/copy.py", line 92, in copy
    rv = reductor(4)
TypeError: cannot pickle 'dict_items' object
```